src/clj/rems/application/model.clj

```diff
@@ -80,6 +80,7 @@
 (defmethod application-base-view :application.event/member-uninvited
   [application event]
   (-> application
+      ;; TODO rename to member-invitations?
```

Comment:
See my previous comment:

> invited actors are stored separately from members because we don't want to leak them via the `:application/invited-members` key

Reply:
But none of the tokens should ever leave the backend? At least there is no reason to give them out now since we are also not showing them to handlers or applicant.

Reply:
Yes, but the values of of the `:application/invitation-tokens` map used in `hide-invitation-tokens` to populate `:application/invited-members`. Exposing invited members to everybody is fine, but we probably don't want to expose invited reviewers/deciders to the applicant.

Reply:
Yes we don't want to expose those invitations. I just think that now we are complicating our internal model with the two. And since we already have custom code to convert one kind of tokens to visible invitations, this same piece of code could rather just handle this concern of selecting the tokens it needs to expose and not expose others? It'd have the role available to select with? Since we also have to dissoc the new actor invitations too this choice is not helping with the information leak either. But I could live with a change where the model has these two separate keys if the naming and handling is symmetrical. And we get a small additional benefit of not confusing applicant side with the handling side so easily. I don't mind breaking this internal part I think.

Reply:
hmm ok, I'll see how storing all invitations under :application/invitation-tokens would look

Reply:
this refactor turned out to be a simplification, thanks for the suggestion